pyproject.toml

```diff
@@ -46,6 +45,9 @@ PyMySQL = ["PyMySQL"]
 [tool.poetry.scripts]
 #flask = 'flask.cli:main'
 
+[tool.poetry.group.dev.dependencies]
```

Comment:
Merge this with the existing dev dependencies key (line 28). If there is a poetry update that changes the key, then we may need to look into that in more detail.

Reply:
According to the [note about the dev-dependencies section](https://python-poetry.org/blog/announcing-poetry-1.2.0/#dependency-groups) the dev-dependencies key is changed from `tool.poetry.dev-dependencies` to `tool.poetry.group.dev.dependencies` and migrating to the new group syntax is recommended.

> As the dev-dependencies is now deprecated, projects should migrate to the new group syntax as soon as possible. Keep in mind that the group syntax is a new feature of Poetry 1.2, and your project will not be buildable with Poetry 1.1 after migrating.

Reply:
Then merge all into the new syntax and add a note in the readme that poetry >= 1.2 is required

Reply:
I migrated to poetry version `>=1.2` in the `main` branch (0ae547fab1b3698596ebac1cc0ba65c3fd269eb7).

Reply:
Then please resolve any open merge conflict. We can merge this branch after that and add the changes to the API (adding new links etc.) in a new PR.